src/components/Note/Note.tsx

```diff
@@ -0,0 +1,49 @@
+
+import React from 'react';
+import RenderRouter from '../RenderRouter/RenderRouter'
+import Amplify from 'aws-amplify';
+import awsconfig from '../../../src/aws-exports';
+
+Amplify.configure(awsconfig);
+
+interface Props {
+  data: any
+  content?: any
+}
+interface State {
+  data: any
+  content: any
+
+}
+export default class VideoPlayer extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      data: props.data,
+      content: this.props.content
+    }
+  }
+  componentDidUpdate(prevProps: Props) {
+    if (this.props.data !== prevProps.data) {
```

Comment:
An easy way to reduce nesting in your function is to invert statement like this. So you end up with
```
if (this.props.data === prevProps.data) {
  return
}
...
```
I recommend using this pattern any time you have a return in one branch, or the function ends.